### PR TITLE
feat(landing-page): add about us

### PR DESCRIPTION
**Pull request by hafidluqman50**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -65,7 +66,7 @@
                         <h1 class="text-center">
                             Jual Laptop & Notebook Terpercaya
                         </h1>
-                        <p>
+                        <p class="mb-5 mt-5">
```
Comment (liannarg66) [inline index.html:69]: add text align: justify

**Review by liannarg66 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (liannarg66) [inline index.html]: add text align: justify

**Review by liannarg66 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (hafidluqman50) [reply index.html]: Done

**Review by liannarg66 — APPROVED** [review]

**Comment by hafidluqman50** [comment]
Bismillah

**Pull request merged**